kahuna/public/js/crop/view.html

```diff
@@ -8,22 +8,22 @@
     <gr-top-bar-actions>
         <div class="top-bar-item side-padded">{{ ctrl.cropWidth() }} &times; {{ ctrl.cropHeight() }}</div>
 
-        <label class="top-bar-item clickable side-padded" title="landscape">
+        <label class="top-bar-item clickable side-padded titip-default titip-bottom" data-title="landscape [l]">
```

Comment:
The latter. Does it work with `shift` or not?

AFAICS in interfaces I use, shortcut hints tend to be capitalised, e.g. `Ctrl+C`, without implying having to press `shift`.